StRoot/StarGenerator/BASE/StarPrimaryMaker.cxx

```diff
@@ -29,12 +29,22 @@ ClassImp(StarPrimaryMaker);
 #include "TMCProcess.h"
 
 #include "tables/St_vertexSeed_Table.h"
+#include <map>
+#include <string>
 
 using namespace std;
 
 // 1 mm / speed of light
 const double mmOverC = 1.0E-3 / TMath::C();
 
+std::map< std::string, std::function< TLorentzVector()> > vertexFunctionMap;
```

Comment:
This map is defined outside of the maker but is initialized in the maker. This is an undefined behavior when two instances are created (as in MT :) ) even though they do the same thing.

Reply:
To be fixed in the next commit / push.